bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotFactory.java

```diff
@@ -118,6 +118,13 @@ public String name() {
             if (repo.value().contains("ignorestale")) {
                 botBuilder.ignoreStaleReviews(repo.value().get("ignorestale").asBoolean());
             }
+            if (repo.value().contains("issuetypes")) {
+                var types = repo.value().get("issuetypes").asArray()
```

Comment:
You just do .stream directly here, right? At least I tend to do that.. :)

Reply:
Right, yes, will fix!